core/src/main/scala/org/broadinstitute/dsde/rawls/expressions/parser/antlr/ErrorThrowingListener.scala

```diff
@@ -29,8 +29,11 @@ class ErrorThrowingListener extends BaseErrorListener {
                            msg: String,
                            e: RecognitionException
   ): Unit = {
-    val errorMsg =
+    val errorMsg = if (msg.startsWith("mismatched input")) {
+      s"Error while parsing the expression. The value you entered is not in the correct format for this data type. For a string, format input as \"value\". For an array, format input as {\"value1\", \"value2\"}. For a file, format input as \"gs://\". For a boolean, format input as `true` or `false`."
```

Comment:
wow, is this correct? Arrays use `{…}` instead of `[...]`?

Reply:
This is what I got from Beth, testing all data types now. But yeah, a lot of users were confused by that error message not knowing what to do about it so trying to make their lives a bit easier

Reply:
Definitely should be [...]!